docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -537,8 +539,6 @@ a temporary ``Apps`` instance::
                     apps = test_apps
             ...
 
```

Comment:
I think this whole section can be reduced to a short explanation, a link to the new reference, and maybe some usage examples in the new docs.

This first example also isn't particularly useful IMO, since we really just want people to use `isolate_apps`, we don't need to explain how it works under the hood. The `Meta.apps` attribute isn't documented.